### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by krolik-exe**
FIRST MERGE #6 , because of the changes that are made in this mentioned pr

### CODE REVIEW

**Comment by krolik-exe** [comment]
This is ready for review too now , because all the changes to readme are done

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
-# spigot-resolver
-small tool to create a wiki page over at spigotmc
+# Spigot-Resolver
 
-wiki page: https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions/
-forum thread: https://www.spigotmc.org/threads/spigot-nms-and-minecraft-version-overview.233194/
+This is a small tool , to create a wiki over at spigotmc.org
```
Comment (MiniDigger) [inline README.md]:
```suggestion
This is a small tool to create a wiki page over at spigotmc.org
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+Versions 1.10 - 1.15 - https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-10-1-15/
+
+Versions 1.16 and up - https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-16/
+
+Forum Thread on spigotmc - https://www.spigotmc.org/threads/spigot-nms-and-minecraft-version-overview.233194/
+
+---------------------------------------------------------------------------------
+## How To: (Build it from source)
+
+To compile Spigot-Resolver , You need:
```
Comment (MiniDigger) [inline README.md]:
```suggestion
To compile Spigot-Resolver you need:
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+
++ Internet connection
++ JDK 8 or newer version of JDK
+
+### You need to :
```
Comment (MiniDigger) [inline README.md]:
```suggestion
### You need to
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+---------------------------------------------------------------------------------
+## How To: (Build it from source)
+
+To compile Spigot-Resolver , You need:
+
++ Internet connection
++ JDK 8 or newer version of JDK
+
+### You need to : 
+Clone this repo , and run this in the project directory:
```
Comment (MiniDigger) [inline README.md]:
```suggestion
Clone this repo and run this in the project directory:
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+Legacy versions 1.8 - 1.9 - https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-legacy/
+
+Versions 1.10 - 1.15 - https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-10-1-15/
+
+Versions 1.16 and up - https://www.spigotmc.org/wiki/spigot-nms-and-minecraft-versions-1-16/
+
+Forum Thread on spigotmc - https://www.spigotmc.org/threads/spigot-nms-and-minecraft-version-overview.233194/
+
+---------------------------------------------------------------------------------
+## How To: (Build it from source)
```
Comment (MiniDigger) [inline README.md]:
```suggestion
## How To (build from source)
```

File: README.md [hunk 6]
```diff
@@ -1,5 +1,32 @@
...
+### You need to : 
+Clone this repo , and run this in the project directory:
+
+```shell
+./gradlew clean build shadowjar
+```
+
+The program should be located in `/libs/build/`
+
+To run this program You need to be in this directory and run from terminal/cmd `java -jar shadow-all.jar`
```
Comment (MiniDigger) [inline README.md]:
```suggestion
To run this program you need to be in this directory and run from terminal/cmd `java -jar shadow-all.jar`
```

**Review by MiniDigger — CHANGES_REQUESTED** [review]

**Comment by MiniDigger** [comment]
thanks

**Pull request merged**